app/Console/Commands/DeleteOldVersions.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use DB;
+use Carbon\Carbon;
+use Auth;
+
+class DeleteOldVersions extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'db:DeleteOld';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Delete versions older than a year';
+
+    /**
+     * Create a new command instance.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        //We must be logged in with root account to execute the function.
+        Auth::attempt(['email'=> 'root@example.com', 'password' => 'rootPassword']);
```

Comment:
Auth::loginUsingId(1);

Its better as in a later version the root user password should be changed